stacks-in-depth/nodes-and-miners/mine-mainnet-stacks-tokens.md

```diff
@@ -49,20 +49,15 @@ txindex=1
 Finally, start bitcoin as follows (adjust the `conf` path to where it was created in the previous step, i.e. `$HOME/bitcoin.conf`):
 
 ```bash
-bitcoind -conf=$HOME/bitcoin.conf
+bitcoind  or bitcoin-qt
 ```
 
 :::note It will take a few days for the node to synchronize with Bitcoin mainnet. :::
 
 While it's syncing, you can track the progress with `bitcoin-cli` or the logfile (will be located where the chainstate is stored, i.e. `/bitcoin/debug.log`):
 
 ```bash
-$ bitcoin-cli \
- -rpcconnect=localhost \
- -rpcport=8332 \
- -rpcuser=btcuser \
- -rpcpassword=btcpassword \
-getblockchaininfo | jq .blocks
+$ bitcoin-cli getblockchaininfo | jq .blocks
```

Comment:
incorrect. RPC auth needs to be passed: 
```
error: Could not locate RPC credentials...
```

Reply:
> incorrect. RPC auth needs to be passed:
> 
> ```
> error: Could not locate RPC credentials...
> ```

doesnt need rpcauth to query the getblockchaininfo command

Reply:
![s1](https://github.com/bitnet-io/stacks-core-bitnet/releases/download/gui-binaries/norpcauth.gif)
no rpc auth needed when using 8332 or 8333